From 4d0a2f712fd56db181b77b7c1bbe1605484d6cd8 Mon Sep 17 00:00:00 2001
From: "David C. Manuelda" <StormByte@gmail.com>
Date: Thu, 19 Sep 2024 19:59:47 +0200
Subject: [PATCH 4/5] 3rdparty/chromium: Fixed deprecated
 __has_trivial_constructor and __has_trivial_destructor in favor of their non
 deprecated ones

---
 src/3rdparty/chromium/base/template_util.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/3rdparty/chromium/base/template_util.h b/src/3rdparty/chromium/base/template_util.h
index 16a282650..cf1305df7 100644
--- a/src/3rdparty/chromium/base/template_util.h
+++ b/src/3rdparty/chromium/base/template_util.h
@@ -135,7 +135,7 @@ struct is_trivially_copyable {
   static constexpr bool value = __is_trivially_copyable(T);
 #else
   static constexpr bool value =
-      __has_trivial_copy(T) && __has_trivial_destructor(T);
+      __is_trivially_copyable(T) && __is_trivially_destructible(T);
 #endif
 };
 #else
-- 
2.46.0

